### PR TITLE
Kernel 21

### PR DESCRIPTION
**Pull request by dlech**

### CODE REVIEW

File: news/_posts/2017-07-25-kernel-release-cycle-21.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+author: "@dlech"
+title: "Kernel Release Cycle 21"
+---
+
+![NXTCam5](/images/mindsensors/nxtcam5.png)
+{: .image-responsive .pull-right}
+
+[mindsensors.com] Has released a new camera sensors, the [NXTCam5]. So, we've added support for it to ev3dev. The sensor API is basically the same
```
Comment (WasabiFan) [inline news/_posts/2017-07-25-kernel-release-cycle-21.md]: `sensors` -> `sensor`

File: news/_posts/2017-07-25-kernel-release-cycle-21.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+as it's predecessors, but this camera is much more advanced. For example, it
```
Comment (WasabiFan) [inline news/_posts/2017-07-25-kernel-release-cycle-21.md]: "it's predecessors" -> `its predecessors'`

**Comment by dlech** [comment]
Preview: http://dlech.github.io/ev3dev.github.io/news/

**Comment by dlech** [comment]
Thanks for being the grammar police.

**Comment by WasabiFan** [comment]
Haha, happy I could be of assistance 😄

**Pull request merged**